test/e2e/tools/tools_test.go

```diff
@@ -48,5 +54,30 @@ var _ = Describe("[tools] Auxiliary tools", func() {
 			_, ok := platform.ParsePlatform(text)
 			Expect(ok).To(BeTrue(), "cannot recognize detected platform: %s", text)
 		})
+
+		It("[api][manager][inspectfeatures] should expose correct active features", Label("api", "inspectfeatures"), func(ctx context.Context) {
+			By("inspect active features from local binary")
+			cmdline := []string{
+				filepath.Join(BinariesPath, "manager"),
+				"-inspect-features",
```

Comment:
nit (please DO NOT resubmit just to fix this): albeit `golang` apps accept both ways, let's use `--double-dashes` for long options and `-s`ingle dash for short options

Reply:
right indeed, I followed the help, I'll fix it

Reply:
right indeed, I followed the help, I'll fix that